hangups/conversation.py

```diff
@@ -11,34 +12,62 @@
 
 
 @asyncio.coroutine
-def build_user_conversation_list(client):
+def build_user_conversation_list(client, conversations_per_request=100):
```

Comment:
So, this is an interesting question. Do we want to do lazy pagination of hangouts, or just load all of them at the start? Lazy pagination of the hangout list would likely take a few more tweaks than the scope of this PR covers, so I'm just inclined to hardcode it. It's fast enough loading a couple hundred hangouts from my tests.